kitchen.docker.sh

```diff
@@ -7,6 +7,12 @@ export KITCHEN_YAML="${THIS_DIR}/kitchen.docker.yml"
 export KITCHEN_GLOBAL_YAML="${THIS_DIR}/kitchen.global.yml"
 export KITCHEN_DRIVER=dokken
 
+# Export DOCKER_HOST variable when using Colima
+if [[ $(command -v colima) ]]; then
+  colima status || colima start
+  export DOCKER_HOST=unix:///Users/$(whoami)/.colima/default/docker.sock
```

Comment:
What about using $HOME, so that it works cross platform?

Reply:
Changed